code/datums/spells/shapeshift.dm

```diff
@@ -1,6 +1,6 @@
 /obj/effect/proc_holder/spell/targeted/shapeshift
-	name = "Shapeshift"
-	desc = "Change shapes"
+	name = "Wild Shapeshift"
```

Comment:
If your gonna keep the robot change the name to: Shapechange.

http://www.d20srd.org/srd/spells/shapechange.htm
D&D memes.